src/plugins/intel_gpu/src/graph/impls/onednn/primitive_onednn_base.h

```diff
@@ -44,11 +48,55 @@ struct typed_primitive_onednn_impl : public typed_primitive_impl<PType> {
           _outer(arg),
           _desc(desc),
           _attrs(attrs),
-          _pd(pd),
-          _prim(pd) { }
+          _pd(pd) {
+            build_primitive();
+        }
 
     bool is_cpu() const override { return false; }
 
+private:
+    std::string get_cache_path() const {
```

Comment:
nit: what about removing this function? we can merge it into `get_cache_filepath` ..

Reply:
There are two api have similar name.
get_cache_path(): It gets cache directory to save cache files.
get_cache_filepath: It generates cache filename from cache-key.

I'll update those name to get_cache_directory() / generate_cache_path_from_key.